src/assets/stylesheets/shared/_custom-variables.scss

```diff
@@ -14,3 +14,4 @@ $color-pf-dark: #363636;
 $pf-icon-2x: 2em;
 $pf-icon-3x: 3em;
 $pf-icon-4x: 4em;
+$font-size-base: 14px;
```

Comment:
@joshuawilson that's odd, as it looks like there is a new line for me when I look at all of the commits combined